lib/identity_cache/query_api.rb

```diff
@@ -108,7 +108,7 @@ def prefetch_associations(associations, records)
 
       def raise_if_scoped
         if current_scope
-          IdentityCache.logger.error("#{name} has scope: #{current_scope.where_values_hash.inspect}")
+          IdentityCache.logger.error("#{name} has scope: #{current_scope.to_sql}")
```

Comment:
I would also include `Shop.current_scope.values.keys` since values like `:preload` wouldn't show up.  Note that it is also possible that `unscoped` is being used, which would result in `Shop.current_scope.values.keys` being empty, although an empty current_scope is still significant since it overrides the default scope.